README.md

```diff
@@ -21,23 +25,31 @@ environments.
 
 ### This Repository
 
-* [C API](./modality-probe-capi): Interact with a Modality probe from C.
-* [CLI](./modality-probe-cli): The CLI used for code generation for
+- [C API](./modality-probe-capi): Interact with a Modality probe from C.
+- [CLI](./modality-probe-cli): The CLI used for code generation for
   probes and the visualization of a trace.
-* [UDP Collector](./collectors/modality-probe-udp-collector): A
+- [UDP Collector](./collectors/modality-probe-udp-collector): A
   UDP-based service that collects probes' outgoing reports.
-* [Debug Collector](./collectors/modality-probe-debug-collector): A
+- [Debug Collector](./collectors/modality-probe-debug-collector): A
   collector that uses JTAG to retrieve data from the probes' logs.
-* [Batch
-  Collector](./collectors/modality-probe-offline-batch-collector): A
+- [Batch Collector](./collectors/modality-probe-offline-batch-collector): A
   utility for converting batches of binary report blobs into log
   files.
 
+
+### Open-source `modality-probe` vs. Commercial [Modality](https://docs.auxon.io/modality/)
+This `modality-probe` repository represents a subset of Modality's toolset.
+- Instrumentation for probes and events are the same in `modality-probe` and Modality.
+-  Modality adds instrumentation for [mutators](https://docs.auxon.io/modality/reference/glossary.html#mutator), which can change system state.
```

Comment:
"Modality adds mutators, allowing you to precisely manipulate system state."

Main point for me is removing "instrumentation". Adding instrumentation by itself isn't interesting, it's the functionality that's interesting, which is achieved with both instrumentation and the backend/CLI/etc... The main point is overall mutation functionality, not just instrumentation.

Reply:
:+1: Going with your exact phrasing.